src/NzbDrone.Core/Download/FailedDownloadService.cs

```diff
@@ -34,7 +34,7 @@ public void MarkAsFailed(int historyId)
             var downloadId = history.DownloadId;
             if (downloadId.IsNullOrWhiteSpace())
             {
-                PublishDownloadFailedEvent(new List<History.History> { history }, "Manually marked as failed");
+                PublishDownloadFailedEvent(new List<Core.History.History> { history }, "Manually marked as failed");
```

Comment:
How about we rename `History.History` to `History.EpisodeHistory`? Avoids the namespace clash and seems more applicable now we have a separate history for downloads.